Demo/src/main/java/com/paypal/android/di/DemoModule.kt

```diff
@@ -15,11 +15,10 @@ object DemoModule {
     @Provides
     fun provideEligibilityAPI(
         coreConfig: CoreConfig
-    ): EligibilityAPI = EligibilityAPI(coreConfig)
+    ): EligibilityAPI = EligibilityAPI(coreConfig, com.paypal.android.BuildConfig.CLIENT_ID)
```

Comment:
+1

Reply:
i dont know what you mean. The `clientId` is requiered for `EligibilityAPI`